README.md

```diff
@@ -65,7 +65,22 @@ Start indexing documents simply by creating or saving them:
 Artist.create(name: 'Frank Stella', keywords: ['art', 'minimalism'])
 ```
 
-Stella adds `after_save` and `after_destroy` callbacks for inline indexing, override these callbacks (namely `#es_index` and `#es_delete`) if you'd like to do your indexing in a background process.
+Stella adds `after_save` and `after_destroy` callbacks for inline indexing, override these callbacks if you'd like to do your indexing in a background process. For example:
+
+```ruby
+class Artist < ActiveRecord::Base
+  include Stella::Searchable
+
+  # disable stella inline callbacks
+  skip_callback(:save, :after, :es_index)
+  skip_callback(:destroy, :after, :es_delete)
+
+  # declare your own
+  after_save :delay_es_index
+  after_destroy :delay_es_delete
```

Comment:
The `searchable` block currently takes a `settings` param, so I'd like to leave that block focused primarily on settings and mappings. I feel it's overloading it by adding async indexing configuration to it. 

One option is to not declare callbacks in this lib at all and leave it up to the client dev to configure those, but I feel they are sensible defaults which allow stella to 'just work' out of the box.

I could add an extra callback registration hook just to manage indexing callbacks, eg `register_search_callback` which would be used something like,

```ruby
register_search_callback :after_save, :my_indexing_method
```

This would ensure only one search callback is called per event. I guess it's a bit cleaner.